spec/features/users/sign_in_spec.rb

```diff
@@ -8,6 +8,12 @@
   include SpAuthHelper
   include IdvHelper
 
+  scenario 'make certain banner shows fake when not in production' do
+    signin('test@example.com', 'Please123!')
+
+    expect(page).to have_content 'FAKE'
+  end
+
```

Comment:
What do you think about using a view spec instead, in `spec/views/layouts/application.html.slim_spec.rb`? That way, we know this will work on all pages as opposed to just the sign in page.

Reply:
Done